pkg/network/node/networkpolicy.go

```diff
@@ -189,10 +189,16 @@ func (np *networkPolicyPlugin) Start(node *OsdnNode) error {
 		return err
 	}
 
-	// Rate-limit calls to np.syncFlows to 1-per-second after the 2nd call within 1
-	// second. The maxInterval (time.Hour) is irrelevant here because we always call
-	// np.runner.Run() if there is syncing to be done.
-	np.runner = async.NewBoundedFrequencyRunner("NetworkPolicy", np.syncFlows, time.Second, time.Hour, 2)
+	// Rate-limit calls to np.syncFlows to once per minSyncPeriod. The maxInterval
+	// (time.Hour) is irrelevant here because we always call np.runner.Run() if there
+	// is syncing to be done.
+	minSyncPeriod := time.Second
+	if np.inMigrationMode {
+		// The existing code can't keep up with the churn rate in large clusters
+		// when in migration mode.
+		minSyncPeriod = 10 * time.Second
```

Comment:
is there a chance of wrongly allowing or denying existing connections for 10s because of this change?

Reply:
So of course, in the affected clusters, this is basically already happening because of slow syncs.

If someone modifies the NetworkPolicies or Pod/Namespace labels, then it will take us up to 10s to catch up to that modification. But there is no reason why someone would need to do that in the middle of a migration, so this doesn't seem like a huge problem.

It does mean that pods will be fully-isolated for up to 10s on startup. It would be good to avoid that, but the goal of this particular PR was to be simple... (I couldn't even come up with a good way to make the behavior be conditional on "it's a big cluster", because this code runs before the informer caches have fully synced, and there's no way to change the parameters of the BoundedFrequencyRunner after creating it.)

Reply:
My concern is that smaller clusters that didn't have this extra delay will now get it, trying to figure out what the result of this could be.
Pods that are migrating to the new node will be ignored for 10s I guess, so if a migrated pod tries to immediately reach a non-migrated pod after its startup, this could fail?

Reply:
yes, though presumably it will retry. But you'll get additional delay, and spurious error messages in your pod logs.